soc/nordic/nrf53/Kconfig

```diff
@@ -165,6 +165,7 @@ config NRF_SPU_RAM_REGION_SIZE
 config SOC_NRF_GPIO_FORWARDER_FOR_NRF5340
 	bool "Forward GPIO pins to network core"
 	depends on NRF_SOC_SECURE_SUPPORTED
+	depends on SOC_NRF53_CPUNET_ENABLE
```

Comment:
I don't think this is correct. One may still want to forward pins to the Network core without enabling it on boot (this can be done later with a call to `nrf53_cpunet_enable()`).

Reply:
is there any Kconfig that enables that API or is it always available?